Makefile

```
@@ -185,7 +185,41 @@ docker-push: ## Push docker image with the manager.
+# OLM operator: the operator embeds the base operand manifests and applies
+# configuration at runtime via its transform layer, driven by the KedaController
+# CR spec. There are no kustomize overlays at deploy time.
+#
+# So to exercise things the same file-based auth code path, we need to find equivalent
+# ways to test the functionality via the KedaController.
+#
+# If the operand/upstream changes their file_auth overlay (config/e2e/file_auth/patch_operator.yml
+# in the keda repo), or otherwise changes their e2e setup in a way that has olm operator 
+# ramifications, we may need to adjust this to stay in sync.
+.PHONY: e2e-cma-setup
+e2e-cma-setup: ## Set up KEDA for CMA e2e tests (called by CI after OLM install)
```

checkmake (0.3.2)

[warning] Target body for "e2e-cma-setup" exceeds allowed length of 5 lines (8).

(maxbodylength)